Altis_Life.Altis/core/init.sqf

```diff
@@ -179,6 +179,16 @@ life_hideoutBuildings = [];
     false
 } count ["gang_area_1","gang_area_2","gang_area_3"];
 
+// -- Disable commanding view
+if (LIFE_SETTINGS(getNumber,"commandview") isEqualTo 1) then {
+    addMissionEventHandler ["EachFrame",{
+        if(cameraView isEqualTo "GROUP") then {
```

Comment:
I do like whitespace. The lack of whitespace here is causing the [build to fail](https://travis-ci.org/AsYetUntitled/Framework/builds/177963696). 

```
ERROR: Possible missing white space after if detected at ../Framework/Altis_Life.Altis/core/init.sqf Line number: 185
```